### ISSUE TITLE
Games with background video files bink (*.bk2) don't start

### ISSUE THREAD

**Issue by GrigoryNot2** [olegos2/mobox#141]
Games, like hitman absolution, Tom Clancy's ghost recon Wildlands, and many others, where menu background is represented by a bink video files (files with extension bk2) can't to start. 
![Screenshot_20240120-195600_TermuxX11](https://github.com/olegos2/mobox/assets/65012726/7ed948bf-9666-4e22-aff7-0427b39db4f8)
 At the same time, hitman absolution starts and works (not perfect) under termux-box.
![Screenshot_20240112-160948_TermuxX11](https://github.com/olegos2/mobox/assets/65012726/f8304175-47b3-4033-8742-5a8e16f82677)
After launching, we see that the game process appears in the task manager. 
![Screenshot_20240120-203337_TermuxX11](https://github.com/olegos2/mobox/assets/65012726/f352e543-8e9f-4222-ac46-a272dfd2dc3d)
In the case of hitman absolution, the settings window opens, where we can configure the game and click the game start button.
Instead of the game window appearing, we can see the process disappear from the task manager after a few seconds.
![Screenshot_20240120-203347_TermuxX11](https://github.com/olegos2/mobox/assets/65012726/8bfed907-4472-47c5-981c-5530fa177711)
I tried all wines available for installation from the script, all dxvk drivers, all combinations of dinarec settings.

**Comment by olegos2** [comment]
k-lite and media foundation doesn't help?

**Comment by GrigoryNot2** [comment]
No, they don't help. Usually the library for this codec is located in the game folder.
![Screenshot_20240121-155626_TermuxX11](https://github.com/olegos2/mobox/assets/65012726/8427b800-a2d5-4d92-8f79-d44ea18e1464)

**Comment by olegos2** [comment]
Then I will test it on my side later
